archetypes/archetype_definition_qby_msp.json

```diff
@@ -4,7 +4,8 @@
             "QBY-Deploy-VM-Backup",
             "QBY-Deploy-DomainJoin",
             "Deploy-VM-Monitoring",
-            "QBY-Deploy-LAW-Agent"            
+            "QBY-Deploy-LAW-Agent",
+            "QBY-Require-Tags"
```

Comment:
```suggestion
```
It is already assigned at root. So we don't need to assign it twice.

Reply:
Fix as suggestion